src/frontend/citizen-portal/src/app/components/update-dispute-contact/update-dispute-contact.component.ts

```diff
@@ -17,14 +18,18 @@ export class UpdateDisputeContactComponent implements OnInit {
   form: DisputantContactInformationFormGroup;
 
   private state: DisputeStore.State;
+  public preferEmail: boolean = true;
 
   constructor(
     private disputeService: DisputeService,
     private store: Store,
+    private route: ActivatedRoute
   ) {
   }
 
   ngOnInit(): void {
+    this.preferEmail = this.route.snapshot.queryParamMap.get('preferEmail') === null ? false : this.route.snapshot.queryParamMap.get('preferEmail') === 'true';
```

Comment:
`this.preferEmail = this.route.snapshot.queryParamMap.get('preferEmail') === 'true' ? true : false;`

Reply:
done.